packages/daemon/src/app/controller/base-controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Context, inject } from 'midway';
```

Comment:
For consistency of names under the controllers, I'd prefer the name `base.ts` instead.

Reply:
+1